### PR TITLE
RSP-1186/1187: Submit and retrieve penalty groups

### PR DESCRIPTION
**Pull request by johnmorleybjss**
Add new endpoints for the creation of penalty groups and retrieval by ID. Penalty groups are persisted to a new `penaltyGroups` table, whereas the penalties they contain go to the original `penaltyDocuments` table.

### CODE REVIEW

File: mock-data/fake-penalty-notice.json [hunk 1]
```diff
@@ -65,7 +67,7 @@
         "nonEndorsableOffence": [
             "INCORRECT USE OF MODE SWITCH - ARTICLE 34(5) EU 165/2014, 27/10/2016"
         ],
-        "penaltyAmount": "50",
+        "penaltyAmount": 80,
```
Comment (ammarhaiderbjss) [inline mock-data/fake-penalty-notice.json]: Has this been agreed with the app?

File: mock-data/fake-penalty-notice.json [hunk 2]
```diff
@@ -37,14 +38,15 @@
...
     "Hash": "6a1e32a2a319c7674fbd83f34cb07b35fdc1cecab261fd02450da821c359d74d",
     "Enabled": true,
     "Offset": 1521331200,
     "Origin": "APP",
     "Value": { 
-        "penaltyType" : "FP",
-        "paymentStatus": "UNPAID",  
+        "penaltyType" : "FPN",
+        "paymentStatus": "UNPAID",
+        "inPaymentGroup": true,
```
Comment (ammarhaiderbjss) [inline mock-data/fake-penalty-notice.json]: Should be 'inPenaltyGroup'

File: mock-data/fake-penalty-notice.json [hunk 3]
```diff
@@ -28,7 +29,7 @@
         "nonEndorsableOffence": [
             "INCORRECT USE OF MODE SWITCH - ARTICLE 34(5) EU 165/2014, 27/10/2016"
         ],
-        "penaltyAmount": "50.00",
+        "penaltyAmount": 50,
         "paymentDueDate": 479945600,
         "officerName": "Sherlock Holmes",
         "officerID": "Blah111",
```
Comment (ammarhaiderbjss) [inline mock-data/fake-penalty-notice.json]: Need to agree this with the app too.

**Comment by johnmorleybjss** [comment]
Agreed with @ammarhaiderbjss we can resolve his comments above at a later stage as we progress in the discovery of the domain. This code is currently on the dev environment, so we're happy to merge to `develop`.

**Pull request merged**